src/Components/ListOfBooks.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const ListOfBooks = ({ item }) => (
+  <div id="listContent flex-col">
+    <div className="booksList">
+      {
+        item.map((book) => (
+          <div key={book.id} className="book-row flex-row">
+            <div className="bookLeft flex-col">
+              <Book
+                id={book.id}
+                title={book.title}
+                category={book.category}
+                author={book.author}
+              />
+              <div className="buttons-left">
+                <button type="button" className="br blueText">Comments</button>
+                <button type="button" className="br blueText">Remove</button>
+                <button type="button" className="blueText">Edit</button>
+              </div>
```

Comment:
- [ ] These buttons should be inside the Book component, this way it will be easier to handle functionality in upcoming milestones.